src/App.tsx

```diff
@@ -1,24 +1,214 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import {
+  getTodos, addTodos, deleteTodos, updateTodoStatus, updateTodoTitle, USER_ID,
+} from './api/todos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo, TodoToSend } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
+import { ErrorMessage } from './types/ErrorMessage';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loadingTodo, setLoadingTodo] = useState([0]);
```

Comment:
```suggestion
  const [loadingTodoIds, setLoadingTodoIds] = useState([0]);
```
As I see you save ids - so name it properly